host-controller/src/main/java/org/jboss/as/domain/controller/logging/DomainControllerLogger.java

```diff
@@ -56,6 +56,12 @@ public interface DomainControllerLogger extends BasicLogger {
      */
     DomainControllerLogger HOST_CONTROLLER_LOGGER = Logger.getMessageLogger(MethodHandles.lookup(), DomainControllerLogger.class, "org.jboss.as.host.controller");
 
+    /**
+     * A logger with the category of {@code org.jboss.as.domain.controller.operations.deployment.warning}.
+     * <strong>Usage:</strong> Use this in code related to deployment's warnings
+     */
+    DomainControllerLogger DEPLOYMENT_WARN_LOGGER = Logger.getMessageLogger(MethodHandles.lookup(), DomainControllerLogger.class, "org.jboss.as.domain.controller.operations.deployment.warning");
+
```

Comment:
We should not create specific root loggers for specific levels. We create root loggers and then we configure the levels from outside, for example in a logger configuration file.

Reply:
I understand, I just wasn't sure how to implement this correctly since the description of the issue was talking about using distinct logger. So, can I just use the ROOT_LOGGER for the domain and DEPLOYMENT_LOGGER for the standalone @yersan ?

Reply:
> I just wasn't sure how to implement this correctly since the description of the issue was talking about using distinct logger for this

... and I did not properly read the Jira description :(

Got it, it was requested to use a different logger so users can turn off the logger under demand.

In such a case, I would suggest using the following (for sure open to any other idea for the name):

"org.jboss.as.server.deployment.runtime.namecheck"
"org.jboss.as.domain.controller.operations.deployment.runtime.namecheck"

Reply:
DEPLOYMENT_NAMECHECK_LOGGER is ok, variable name is no an issue, if you read it and identify its meaning, that's ok